script/Ciel/Day 7/07_01_CIEL07_1B.txt

```diff
@@ -104,30 +103,31 @@ Weren't you the one who said that cool line about leaving the city after the vam
 [sha:5c6d5310684d37fd6fe9e69d87c356318a4569fe]{
 -- Page 39, Offset 14902.
 -- 「おお。そんな話もありましたね。
+// He didn't say that last line out loud? Does the mind reading come with the mind erasing?
 "Oh, right, I did say something like that.
 }
 [sha:4a714c76068d700ea13ccf61a5ae1d0bf72e439e]{
 -- Page 39, Offset 14903.
 -- あんな状況だったのにしっかり覚えているなんて、中々の記憶力と言わざるをえません。先輩として鼻が高いです。と言いつつ、さりげなく眼鏡のズレを直すわたしなのでした」
-To think you'd remember despite being in such a situation.
- I have to admit your memory is quite impressive.
+To think you'd remember something like that despite the circumstances.
+ I have to admit that your memory is quite impressive.
  Your Senpai is quite proud of you.
```

Comment:
I know I'm the one who wrote this, but the repetition of 'quite' is probably a bit amateurish

Reply:
just "[Your] Senpai is proud of you" is fine